src/main/resources/tags/errors/logintoxbox.tag

```diff
@@ -0,0 +1,12 @@
+type: text
+issues: Please log into Xbox to join this server.
+
+---
+
+This error occurs when Geyser is unable to validate your login. This might be because the client didn't send the right data, or you just aren't logged in.
+You can use the following steps to debug this:
+- Double check you are actually logged in.
+- Try joining a featured server and see if you face the same issue, if you do then continue through the steps.
```

Comment:
if they don't? :p

I'd move this step to after restarting game/device, and tell them to contact Minecraft support if they can't login to featured servers